CS/3-Draw_a_Custom_Signature/README.md

```diff
@@ -5,9 +5,50 @@ This project uses the [.NET MAUI Community Toolkit DrawingView](https://learn.mi
 
 <img src="./media/drawable_signature.gif" alt="drawing" width="300"/>
 
+## Implementation Details
+
 > **Note**
 >
-> The Universal Subscription or an additional Office File API Subscription is required to use this example in production code. Please refer to the following page for pricing information: [DevExpress Subscription](https://www.devexpress.com/Subscriptions/).
+> PDF-related functionality is included in our [Office File API](https://www.devexpress.com/products/net/office-file-api/) subscription.
+>
+> To run this example, you need any of the following subscriptions: [DevExpress Universal](https://www.devexpress.com/subscriptions/universal.xml), [DXperience](https://www.devexpress.com/subscriptions/dxperience.xml), [Office File API](https://www.devexpress.com/products/net/office-file-api/), [WinForms](https://www.devexpress.com/products/net/controls/winforms/), [WPF](https://www.devexpress.com/products/net/controls/wpf/), or [ASP.NET](https://www.devexpress.com/subscriptions/asp/).
+
+The [PdfDocumentProcessor](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentProcessor) class includes APIs that allow you to manipulate PDF files.
+
+To open and sign a PDF file in a .NET MAUI application, you should copy the PDF and PFX certificate files from the application bundle to a device folder:
+
+```csharp
+public async Task<string> CopyWorkingFilesToAppData(string fileName) {
+    using Stream fileStream = await FileSystem.Current.OpenAppPackageFileAsync(fileName);
+    string targetFile = Path.Combine(FileSystem.Current.AppDataDirectory, fileName);
+    using FileStream outputStream = File.OpenWrite(targetFile);
+    fileStream.CopyTo(outputStream);
+    return targetFile;
+}
+```
+
+After that, you can use the [PdfDocumentProcessor.LoadDocument](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentProcessor.LoadDocument.overloads) method to open the PDF file.
+
+This project calls the [PdfAcroFormFacade.GetFields](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfAcroFormFacade.GetFields) method to find all the PDF file fields. Once the fields are found, use the [PdfFormFieldFacade.Type](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfFormFieldFacade.Type) property to get only signature fields. The project signs the first found signature field. 
```

Comment:
First the project is the subject (the project does something), then you go to imperative mood (you do something). Either everything should be instructions, or everything should be a description of how it works. 

```suggestion
This project uses the following members to find the first available signature field: 

* The [PdfAcroFormFacade.GetFields](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfAcroFormFacade.GetFields) method finds all fields in the PDF file. 
* The [PdfFormFieldFacade.Type](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfFormFieldFacade.Type) property helps locate signature fields.  
```